### PR TITLE
chore: initial repo setup 

### PR DESCRIPTION
**Pull request by toddbaert**

### CODE REVIEW

File: .github/.github/workflows/release-please.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.GITHUB_TOKEN}}
+          default-branch: main
+
+      # The logic below handles the npm publication:
```
Comment (justinabrahms) [inline .github/.github/workflows/release-please.yml:19]: Maven instead of npm?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "java.configuration.updateBuildConfiguration": "interactive"
```
Comment (justinabrahms) [inline .vscode/settings.json:2]: Add to git ignore?

**Review by justinabrahms — APPROVED** [review]

File: .github/.github/workflows/release-please.yml (hunk 1 above)
Comment (toddbaert) [reply .github/.github/workflows/release-please.yml:19]: lol nice catch.

File: .vscode/settings.json (hunk 2 above)
Comment (toddbaert) [reply .vscode/settings.json:2]: sure!

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**